scripts/validateSetup.js

```diff
@@ -2,10 +2,7 @@ const Firestore = require('../utils/firestore.js')
 const config = require('config')
 
 // check whether github credentials are not falsy
-if (
-  config.githubOauth.clientId === '<clientId>' ||
-  config.githubOauth.clientSecret === '<clientSecret>'
-) {
+if (config.githubOauth.clientId === '<clientId>' || config.githubOauth.clientSecret === '<clientSecret>') {
   throw new Error('Github credentials are not set.')
 } else {
   // eslint-disable-next-line no-console
```

Comment:
Instead of disabling linting errors for `console.log` for each line, would be better if disable it for the whole file at the top. It would help in reading the code easily
Refer: https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments